docs/bsp.md

```diff
@@ -869,19 +1031,27 @@ trait TestResult {
 
 The field `data` may contain test-related language-specific information.
 
-Notification:
+#### Test Notifications
 
-* method: `buildTarget/testReport`
-* params: `TestReport` defined as follows:
+The beginning of a testing unit may be signalled to the client with a `build/taskStart` notification.
+When the testing unit is a build target, the notification's `dataKind` field should be `test-task` and the 
+`data` field should include a `TestTask` object.
+
+```scala
```

Comment:
I would make stress that a possible use case here for BSP clients is to use the parent-child relationships of task notifications to do a nice tree-rendering of test executions, provided that the build tool respects this hierarchy when sending the task notifications.